pages/new-mission.vue

```diff
@@ -0,0 +1,486 @@
+            </b-form-group>
+          </b-col>
+
+          <b-col>
+            <b-form-group id="TeamSizeGroup">
+              <b-form-input
+                id="newMissionTeamSize"
+                v-model="$v.form.teamSize.$model"
+                placeholder="teamSize"
+                name="newMissionTeamSize"
+                type="number"
+                :state="$v.form.teamSize.$dirty ? !$v.form.teamSize.$error : null"
+                aria-describedby="newMissionTeamSize-live-feedback"
+              />
+
+              <b-form-invalid-feedback id="newMissionTeamSize-live-feedback" />
+            </b-form-group>
+          </b-col>
+
+          <b-col>
+            <b-form-group id="locationGroup">
+              <b-form-input
+                id="newMissionLocation"
+                v-model="$v.form.location.$model"
+                placeholder="location"
+                name="newMissionLocation"
+                type="text"
+                :state="$v.form.location.$dirty ? !$v.form.location.$error : null"
+                aria-describedby="newMissionLocation-live-feedback"
+              />
+
+              <b-form-invalid-feedback id="newMissionLocation-live-feedback" />
+            </b-form-group>
+          </b-col>
+        </b-row>
+
+        <b-row>
+          <b-col>
+            <v-select v-model="list"
+                      label="name"
+                      taggable
+                      multiple
+                      :options="environments"
+                      @input="addToDrop"
+            />
+          </b-col>
+
+          <b-col>
+            <draggable
+              tag="transition-group"
+              :component-data="componentData"
+              :list="list"
+              class="list-group"
+              draggable=".item"
+              :animation="100"
+              @start="dragging = true"
+              @end="dragging = false"
+            >
+              <div
+                v-for="element in list"
+                :key="element.name"
+                class="list-group-item item"
+              >
+                <b-row style="align-items: center">
+                  <b-col style="display: flex; align-content: center; align-items: center;">
+                    <img class="round--picto"
+                         :src="element.logo"
+                    >
+                    <p>{{ element.name }}</p>
+                  </b-col>
+                </b-row>
+              </div>
+            </draggable>
+          </b-col>
+        </b-row>
+
+        <b-button variant="primary" :disabled="$v.form.$invalid" @click="createMission">
+          Submit
+        </b-button>
+      </b-form>
+    </b-container>
+  </div>
+</template>
+
+<script>
+  import { validationMixin } from 'vuelidate'
+  import { url } from 'vuelidate/lib/validators'
+  import draggable from 'vuedraggable'
+  import 'vue-select/dist/vue-select.css'
+
+  let id = 1
```

Comment:
Is it used ? Or rename it so we can understand this is the id of which element ?